src/ui/pages/NotificationDetails/components/MultiSigRequest/ErrorPage.tsx

```diff
@@ -119,9 +135,12 @@ const ErrorPage = ({
             {i18n.t("notifications.details.identifier.errorpage.help.title")}
           </h2>
           <IonText className="detail-text">
-            {i18n.t(
-              "notifications.details.identifier.errorpage.help.detailtext"
-            )}
+            <Trans
+              i18nKey={i18n.t(
+                "notifications.details.identifier.errorpage.help.detailtext"
+              )}
+              components={[<HandleDiscordLink key="" />]}
```

Comment:
Why we need empty `key` here ?

Reply:
It's a mandatory requirement of the `Trans` component, unfortunately it has to be there, even if it's empty.